addons/stock_account/models/stock_move.py

```diff
@@ -263,8 +263,7 @@ def _action_done(self, cancel_backorder=False):
                 stock_valuation_layers |= getattr(todo_valued_moves, '_create_%s_svl' % valued_type)()
                 continue
 
-
-        for svl in stock_valuation_layers:
+        for svl in stock_valuation_layers.with_context(active_test=False):
```

Comment:
Shouldn't this be interesting to have a comment on why the `active_test` for avoiding latter removal of it?

Reply:
we're removing it in saas13.3, i already added a message on our internal task
this is ugly and we want to remove it asap :)